src/partner-access/partner-access.service.ts

```diff
@@ -9,6 +9,13 @@ import { PartnerAccessCodeStatusEnum } from '../utils/constants';
 import { CreatePartnerAccessDto } from './dtos/create-partner-access.dto';
 import { PartnerAccessRepository } from './partner-access.repository';
 
+// TODO storing base service minimum here but this might need to be a config setup eventually
```

Comment:
Makes sense - I wonder if different partners will want different base access as well. Definitely one for the future